Artesian/Artesian.SDK.Tests/ContentNegotiationTests.cs

```diff
@@ -143,11 +143,11 @@ await qs.CreateActual()
                         .ExecuteAsync().ConfigureAwait(false);
                 });
 
-                Assert.That(ex.Message, Does.Contain("Bad Request"));
-                Assert.That(ex.ProblemDetail, Is.Not.Null);
-                Assert.That(ex.ProblemDetail.Title, Is.EqualTo("Bad Request"));
-                Assert.That(ex.ProblemDetail.Status, Is.EqualTo(400));
-                Assert.That(ex.ProblemDetail.Detail, Is.EqualTo("Invalid request parameters"));
+                Assert.That(ex!.Message, Does.Contain("Bad Request"));
```

Comment:
instead of adding null forgiving, add assertions over ex being not null, and ProblemDetails being not null

apply the same in other places